postgres/defaults.yaml

```diff
@@ -15,12 +15,13 @@ postgres:
   group: postgres
 
   prepare_cluster:
-    command: initdb --pgdata=/var/lib/pgsql/data
-    test: test -f /var/lib/pgsql/data/PG_VERSION
+    pgcommand: initdb --pgdata=
```

Comment:
Having `initdb -D` here will simplify your final command for the `postgresql-cluster-prepared` state.

Reply:
Cool!! I'm only realizing now `initdb -D` and `initdb --pgdata=` are equilivent. ;-)